geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ConnectCommand.java

```diff
@@ -94,9 +94,7 @@ public Result connect(
       @CliOption(key = {CliStrings.CONNECT__USE_HTTP}, specifiedDefaultValue = "true",
           unspecifiedDefaultValue = "false",
           help = CliStrings.CONNECT__USE_HTTP__HELP) boolean useHttp,
-      @CliOption(key = {CliStrings.CONNECT__URL},
```

Comment:
so now, user has to specify either locator, jmxManager or a url. Should we add validation saying that one of them has to be present?

Reply:
Actually `connect`, without any options, will still try and connect to the locator on localhost.